### PR TITLE
Added Fromversion Banner

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/30648

## Description
Added banner to documentation, where written : "Supported Cortex XSOAR versions: {from_version} and later."

### CODE REVIEW

File: content-repo/gendocs.py [hunk 1]
```diff
@@ -127,6 +127,13 @@ def get_deprecated_data(yml_data: dict, desc: str, readme_file: str):
     return ""
 
 
+def get_fromversion_data(yml_data: dict):
+    from_version = yml_data.get('fromversion')
+    if from_version:
+        return f':::info beta\nSupported Cortex XSOAR versions: {from_version} and later.:::\n\n'
```
Comment (ShahafBenYakir) [inline content-repo/gendocs.py:133]:
```suggestion
        return f':::info beta\nSupported Cortex XSOAR versions: {from_version} and later.\n:::\n\n'
```

File: content-repo/gendocs.py [hunk 2]
```diff
@@ -127,6 +127,13 @@ def get_deprecated_data(yml_data: dict, desc: str, readme_file: str):
     return ""
 
 
+def get_fromversion_data(yml_data: dict):
+    from_version = yml_data.get('fromversion', '')
+    if not from_version.startswith(('4', '5.0')):
```
Comment (glicht) [inline content-repo/gendocs.py:132]: Also check that from_version is not empty

File: content-repo/gendocs_test.py [hunk 3]
```diff
@@ -229,6 +229,13 @@ def test_get_deprecated_data():
     assert "Add information" not in res
 
 
+def test_get_fromversion_data():
+    res = get_fromversion_data({'fromversion': '5.5.0'})
+    assert 'Supported Cortex XSOAR versions: 5.5.0 and later.' in res
+    res = get_fromversion_data({'fromversion': '5.0.0'})
+    assert 'Supported Cortex XSOAR versions: 5.0.0 and later.' not in res
```
Comment (glicht) [inline content-repo/gendocs_test.py:236]: Better to assert that the result is an empty string.

File: content-repo/gendocs.py (hunk 2 above)
Comment (daryakoval) [reply content-repo/gendocs.py:132]: done

File: content-repo/gendocs_test.py (hunk 3 above)
Comment (daryakoval) [reply content-repo/gendocs_test.py:236]: done

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**